packages/wrap/src/transforms/FilterFieldDirectives.ts

```diff
@@ -0,0 +1,39 @@
+import { GraphQLSchema, GraphQLFieldConfig } from 'graphql';
+import { Transform, getDirectives } from '@graphql-tools/utils';
+import TransformObjectFields from './TransformObjectFields';
+
+export default class FilterFieldDirectives implements Transform {
+  private readonly filter: (dirName: string, dirValue: any) => boolean;
+
+  constructor(filter: (dirName: string, dirValue: any) => boolean) {
+    this.filter = filter;
+  }
+
+  public transformSchema(originalSchema: GraphQLSchema): GraphQLSchema {
+    const transformer = new TransformObjectFields(
+      (_typeName: string, _fieldName: string, fieldConfig: GraphQLFieldConfig<any, any>) => {
+        const valueMap = getDirectives(originalSchema, fieldConfig);
```

Comment:
getDirectives could return an array for a given directive if the directive is repeated, but you would want to remove only the directive with the matching value